app/controllers/home_controller.rb

```diff
@@ -1,2 +1,13 @@
 class HomeController < ApplicationController
+  def index
+
+  end
+
+  def admin
```

Comment:
You don't need it here. All functionality regarding to admin part should be in admin controller